dll/win32/shell32/folders/CControlPanelFolder.cpp

```diff
@@ -488,8 +488,8 @@ HRESULT WINAPI CControlPanelFolder::GetUIObjectOf(HWND hwndOwner,
 */
 HRESULT WINAPI CControlPanelFolder::GetDisplayNameOf(PCUITEMID_CHILD pidl, DWORD dwFlags, LPSTRRET strRet)
 {
-    if (!pidl)
-        return S_FALSE;
+    if (!strRet || !pidl)
+        return E_INVALIDARG;
```

Comment:
Are there other GetDisplayNameOf functions in our code base that may require such a similar fix?

Reply:
Not that I spotted in a brief sweep.

Reply:
the PIDL return value here is actually correct, passing in a null pidl is not an error but a request to get the item name itself.